### PR TITLE
fix: support multi user notification (AR-2797)

### PR DESCRIPTION
**Pull request by MohammedMokresh**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2797" title="AR-2797" target="_blank">BugAR-2797</a>  Notifications are not received for first logged in account
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

when a session is not active, the user won't receive a notification on that account

### Solutions
Remove the check if the session is active, add userId to the notification channel, and create a group channel for each logged in user.
Also group the notifications for different users and mark the groups to determine for which user is it. 

### Attachments (Optional)

Wire 2022-12-12 at 17_23 copy

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1215/1/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5196198)(5196198)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5196198)(5196198)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/notification/NotificationChannelsManager.kt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+    /**
+     * Tricky bug: No documentation whatsoever, but these values affect how the system cancels or not the vibration of the notification
+     * on different Android OS levels, probably channel creation related.
+     */
+    private fun getAudioAttributeUsageByOsLevel() =
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) AudioAttributes.USAGE_NOTIFICATION_RINGTONE else AudioAttributes.USAGE_MEDIA
+
+    companion object {
+        private fun getChanelGroupNameForUser(userName: String): String = userName
```
Comment (alexandreferris) [inline app/src/main/kotlin/com/wire/android/notification/NotificationChannelsManager.kt:134]: Is this needed? 🤔 It has only one usage and it simply returns what it received or am I missing something?

**Review by alexandreferris — COMMENTED** [review]
Great work here!

I just have one tiny question 🤔

**Review by alexandreferris — APPROVED** [review]
🚀

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1215/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5212813)(5212813)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5212813)(5212813)-dev-release.apk)

**Pull request merged**